content/en/hub/initial-setup/install/getting-started/index.md

```diff
@@ -50,38 +50,60 @@ your system and boot from the CD.
 
 ### Windows
 
-To write the FreeNAS installer to a USB on Windows, plug the USB into
-the system and use [Image Writer](https://launchpad.net/win32-image-writer/)
-or [Rufus](http://rufus.akeo.ie/).
+To write the TrueNAS installer to a USB stick on Windows, plug the USB stick into
+the system and use [Rufus](http://rufus.akeo.ie/) or [Balena Etcher](https://www.balena.io/etcher/). 
```

Comment:
Thanks for the PR!
We would rather not recommend Etcher.  Not only does it do things differently and sometimes result in non-booting USB sticks. It can, depending on the brand of USB sticks, completely ruin the USB stick.  I've personally lost two USB sticks due to using Etcher. 
Otherwise your PR looks good.  If you can clean that bit up, we'll merge it in.

Reply:
Good to know, change made. I've changed the recommendation to Rufus with a mention of Win32 Image Writer and its limitations, because we've seen people on the forums get confuzzled when they use Image Writer to write successfully, but then cannot write a second time and don't know how to wipe the stick to get it back to being visible by Windows and thus Image Writer.